packages/openneuro-app/src/scripts/dataset/fragments/dataset-history.jsx

```diff
@@ -80,7 +79,6 @@ const DatasetHistory = ({ datasetId }) => {
                 </div>
                 <div className="col-lg col col-2 grid actions">
                   <Revalidate datasetId={datasetId} revision={commit.id} />
-                  <UpdateRef datasetId={datasetId} revision={commit.id} />
```

Comment:
The mutation it uses hasn't worked since 2022. It would move the HEAD reference in MongoDB but we no longer do that, always following the git repo HEAD reference instead. The git version would be more destructive and I think it's better handled via the git interface itself.

Reply:
Do we allow force-pushes to revert? If not, then it's just logging into the worker and manually doing the job.

Anyway, if it hasn't worked, then this is no loss, so no further objections.